### ISSUE TITLE
Combobox mixed ARIA spec conflict with JAWS

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#2971]
#2756 should be reverted. There are no issues with some other screen readers, but JAWS does not allow for double combobox ARIA role with robustness.

To remain aligned with SLDS, the Combobox should be reverted to the ARIA 1.1 spec and not be the ARIA 1.2 spec.
